file-formats/dsfi/type/zif_aff_dsfi_v1.intf.abap

```diff
@@ -2,18 +2,36 @@ INTERFACE zif_aff_dsfi_v1
   PUBLIC.
 
 
-  "! <p class="shorttext">Engine</p>
-  "! Engine
-  "! $values {@link zif_aff_dsfi_v1.data:co_engine}
-  TYPES: ty_engine TYPE zif_aff_types_v1=>ty_object_name_30.
+  TYPES:
+    "! <p class="shorttext">Engine</p>
+    "! Engine
+    "! $values {@link zif_aff_dsfi_v1.data:co_engine}
+    "! $default {@link zif_aff_dsfi_v1.data:co_engine.analytical_engine}
+    ty_engine TYPE zif_aff_types_v1=>ty_object_name_30.
 
   CONSTANTS:
     BEGIN OF co_engine,
       "! <p class="shorttext">Analytical Engine</p>
       "! Analytical engine
       analytical_engine TYPE ty_engine VALUE 'ANA',
+      "! <p class="shorttext">SQL Engine</p>
+      "! SQL engine
+      sql_engine TYPE ty_engine VALUE 'SQL',
     END OF co_engine.
 
+  TYPES:
+    BEGIN OF ty_sql_properties,
+      "! <p class="shorttext">AMDP Reference</p>
+      "! The name of the AMDP implementation reference
+      "! $required
+      amdp_name          TYPE c LENGTH 100,
```

Comment:
Happy to arrive at the details :)

We unfortunately cannot justify any change that would bring this to a match.

This object type has already been released, thus "AMDP Reference" is already present in the documentation (e.g. Help Portal). Updating the documentation would be onerous and we risk an incomplete update, which could leave the documentation inconsistent.

amdp_name is the technical name of this field across the system, even in contexts unrelated to CDS scalar function implementations. Changing this even just on the interface level would introduce an inconsistency in the standard code.

Reply:
Just to be clear. I think this is a minor from my point of view, because one can map both terms easily. 

I can understand that you want to keep "AMDP Reference" (especially, if this is the terminology). This sounds reasonable to me.

I just want to point out that the name of the AFF interface does not need to match the name ob your persistency for example. The AFF and its interface its meant to map exactly from internal naming to external naming.

Reply:
Thanks for the additional perspective, but as a team we'd prefer to keep the technical names aligned between the persistency and the interface.

Reply:
As discussed, we've agreed to match the field names to their descriptions.

Reply:
Thanks 👍